power_grid_model_c/power_grid_model/include/power_grid_model/container.hpp

```diff
@@ -110,6 +110,19 @@ class Container<RetrievableTypes<GettableTypes...>, StorageableTypes...> {
         assert(is_base<Gettable>[idx_2d.group]);
         return (this->*(func_arr[idx_2d.group]))(idx_2d.pos);
     }
+
+#ifdef _DEBUG
```

Comment:
Only `NDEBUG` is standard:

https://stackoverflow.com/questions/2290509/debug-vs-ndebug